docs/developer/price-expiration.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Price Expiration
+sidebar_label: Price Expiration
+---
+
+:::info
+This feature was introduced in **Saleor 3.21**. In previous versions, the base prices were refreshed with every single recalculation.
+:::
+
+## Introduction
+When adding a line to an order, the product variant price is taken from the relevant channel listing. 
+However, throughout the order lifecycle, price recalculations may be required, and channel listing prices can be updated in the meantime.
+To ensure price consistency over time, regardless of channel listing updates, Saleor provides a settings that allows to control the price freeze period.
+
+### Settings
+You can configure the duration after which a price is considered outdated and requires refreshing from the channel listings. 
+This is managed through the [`channel`](developer/channels/configuration.mdx) setting `draftOrderLinePriceFreezePeriod`, 
+which defines the time in hours after which the order line price will be refreshed. The default value is 24 hours. 
+To prevent price refreshing entirely, set this value to `null` or `0`.
+
+### How It Works
+The price expiration mechanism applies only to orders with `DRAFT` status. Orders in any other status will retain their original prices permanently.
+A new price expiration time is set on an order line when:
```

Comment:
Do not use `new` - how long it's gonna be `new`? Will you remember to change it after it won't being `new`? 😄 
```suggestion
The price expiration time is set on an order line when:
```

Reply:
corrected